lang/fpc/Portfile

```diff
@@ -96,7 +96,7 @@ subport "${name}-cross" {
     worksrcdir      ${name}build-${version}/fpcsrc/compiler
     use_configure   no
     build.args      OPT="-ap -v0"
-    build.target    arm avr i386 i8086 jvm m68k mips mipsel powerpc powerpc64 sparc
+    build.target    aarch64 arm avr i386 i8086 jvm m68k mips mipsel powerpc powerpc64 sparc
```

Comment:
Doesn't adding a build target change what gets built and installed? If yes, then the revision of this subport needs to be increased. I assume that the other changes in this commit are also not merely cosmetic and therefore also would warrant a revision increase to whatever subport(s) they affect.

Reply:
Correct. This adds another crosscompiler. I will try to resolve the issue with building the utilities by increasing the verbosity in the next commit.